.circleci/config.yml

```diff
@@ -355,19 +355,28 @@ jobs:
 
   # Runs backend unit tests and reports coverage to codecov.io.
   backend unit test:
-    docker:
-      - image: node:14
-        auth:
-          username: $DOCKER_USERNAME
```

Comment:
Is there a reason why you removed the Docker auth?

Reply:
I moved it down to line 368.  The test needs to do some things to the containers and then run something.  This isn't just a DC up process anymore.